### ISSUE TITLE
Update plyr version

### ISSUE THREAD

**Issue by eugenelandau** [redxtech/vue-plyr#56]
Update plyr version up to 3.5.2

Current version doesn't work in IE 11

I think this is fixed in version 3.5.1: https://github.com/sampotts/plyr/blob/master/changelog.md#v351

**Comment by stefusilviu** [comment]
Until then, you can add these pollyfils to your project:
```
import 'custom-event-polyfill';
import 'url-polyfill';
```

It worked for me.

**Comment by redxtech** [comment]
I just pushed `v5.1.2` which should have the latest plyr version installed.

I am now getting a message from github saying that there's a vulnerability installed (`merge@1.2.0`). I have traced this issue to `@vue/cli-plugin-unit-jest` not requiring the latest version of `jest`. Updating jest to at least `v24.0.0` bumps `jest-cli` to the same version, which bumps `jest-haste-map` to the same version as well, which bumps `sane` to `v3.0.0`, which bumps `exec-sh` to `v0.3.2.`, which cuts out `merge` as a dependency, getting rid of the issue altogether.

Just have to wait until this happens I guess.

In other words - your issue should be fixed.
